lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -105,6 +105,20 @@ function requestHandler(request, response) {
         }
       }
 
+      if (params.has('extra_cookie')) {
```

Comment:
this file is for our smoketests, we'll want to make sure we add a smoketest that exercises this functionality which is probably going to be tricky...

maybe hold off on this step until we get buy-in from rest of LH team for this feature to work this way?

Reply:
any reason to keep the changes to this file?

Reply:
oh @RynatSibahatau think you could add a smoketest exercising this and our cookies logic?

Reply:
> oh @RynatSibahatau think you could add a smoketest exercising this and our cookies logic?

isn't that going to be kind of hard, @patrickhulce? What would we be testing, exactly?

Reply:
I was thinking a page has a link or something that contains the contents of the cookies and it gets picked up by the anchor elements artifact we can assert